cmd/atlas/cmd/apply.go

```diff
@@ -0,0 +1,39 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+type tApplyFlags struct {
```

Comment:
this `t<Something>` , not very common in Go. You can use an anonymous struct since it's not used anywhere else.

Reply:
👍